source/components/Header/AccountHeader.tsx

```diff
@@ -4,28 +4,144 @@ import React, { useEffect } from 'react';
 import { useSelector } from 'react-redux';
 import { browser } from 'webextension-polyfill-ts';
 
-import { IconButton, Icon } from 'components/index';
+import { IconButton, Icon, Tooltip } from 'components/index';
 import { useUtils } from 'hooks/index';
 import { RootState } from 'state/store';
 import { getController } from 'utils/browser';
 import { ellipsis } from 'utils/index';
 
+type RenderAccountsListByBitcoinBasedProps = {
+  setActiveAccount: (id: number) => Promise<void>;
+};
+
+const RenderAccountsListByBitcoinBased = (
+  props: RenderAccountsListByBitcoinBasedProps
+) => {
+  const { setActiveAccount } = props;
```

Comment:
if SetActiveAccount is used just inside `RenderAccountsListByBitcoinBased` you can create this fn inside `RenderAccountsListByBitcoinBased` instead of passing by props